kinder/pkg/cluster/manager/actions/kubeadm-init.go

```diff
@@ -209,24 +201,10 @@ func kubeadmInitWithPhases(cp1 *status.Node, automaticCopyCerts bool, kustomizeD
 
 	if automaticCopyCerts {
 		uploadCertsArgs := []string{
-			"init", "phase", "upload-certs",
+			"init", "phase", "upload-certs", "--upload-certs",
 			fmt.Sprintf("--config=%s", constants.KubeadmConfigPath),
 			fmt.Sprintf("--v=%d", vLevel),
 		}
-		if automaticCopyCerts {
```

Comment:
this seems to be checked already above.
just appending the flag --upload-certs to uploadCertsArgs seems sufficient here.